csharp/dotnet-windows/console/Samples/Samples/Program.cs

```diff
@@ -30,7 +30,7 @@ static void Main(string[] args)
                 Console.WriteLine(choose);
                 Console.WriteLine("");
                 Console.WriteLine("1. Pronunciation Assessment with Content.");
-                Console.WriteLine("2. Pronunciation Assessment with Scenario ID.");
+                Console.WriteLine("2. Pronunciation Assessment with Scenario Id.");
```

Comment:
Here "ID" is in a sentence and is not a variable or function name. So please do not change "ID" to "Id".

Reply:
udpated